iiod/responder.c

```diff
@@ -995,7 +991,7 @@ static void handle_free_evstream(struct parser_pdata *pdata,
 				 const struct iiod_command *cmd,
 				 struct iiod_command_data *cmd_data)
 {
-	struct evstream_entry *entry, *each;
+	struct evstream_entry *entry;
```

Comment:
Odd, I would expect the build (or least CI) to catch this...

Reply:
Yah, I was thinking the same thing, but didn't debug it. Just figured it out. I'll push another commit to this PR